src/components/One/index.jsx

```diff
@@ -1,5 +1,30 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import "../../globals.css";
+
 function Index() {
-  return <div>Index</div>;
+  return (
+    <div className="flex flex-col items-center justify-center  text-white  bg-[#141619] min-h-[80px]  px-48 pt-[100px] pb-[128px] relative">
+      <div className="flex flex-col text-center justify-center font-medium leading-[78px] text-[56px] ">
+        Contact Us
+      </div>
+      <div className="flex text-center justify-center text-[#E5E5E5] text-[18px] font-normal leading-[30px] w-full px-[10%] pt-[60px] pb-[62.39px] z-50">
+        We are always open to discuss new value-adding partnerships. Do reach
```

Comment:
try to map values, no need to hardcode anything refer discord chats for more info